packages/yard-editor/dev-webapp/App.tsx

```diff
@@ -76,7 +76,7 @@ export const App = () => {
   }, [editor]);
 
   const onSetContent = useCallback((path: string, content: string) => {
-    const match = /\.sw\.(json|yml|yaml)$/.exec(path.toLowerCase());
+    const match = /\.sw\.(yml|yaml)$/.exec(path.toLowerCase());
```

Comment:
Not related to your changes but this should be `yard` not `sw`, right?
```suggestion
    const match = /\.yard\.(yml|yaml)$/.exec(path.toLowerCase());
```

Reply:
@caponetto Good point, I think `sw` is a bug in the old code that was copied from serverless workflow.

Reply:
Makes sense! Thanks for the fix :)